src/mono/mono/mini/aot-compiler.c

```diff
@@ -448,6 +448,7 @@ static MonoAotCompile *llvm_acfg;
 static MonoAotCompile *current_acfg;
 static MonoAssembly *dedup_assembly;
 static GHashTable *dedup_methods;
+static GPtrArray *dedup_methods_list;
```

Comment:
How do you plan to use the data saved in this array? Do you want to print it out in certain verbose mode or output it in a file by setting a switch?

Reply:
Its just used to iterate over the methods in a fixed order. The hashtable iteration order depends on the hash codes of MonoMethod*'s which are not stable.